### DISCUSSION TITLE
Sequent calculus proofs

### DISCUSSION THREAD

**Discussion by AurelWeinhold** [typst/typst#2514] [Q&A] [answered]
Hi community,

has anyone here ever done sequent calculus proofs in typst, such as those found on the [wikipedia sequent calculus](https://en.wikipedia.org/wiki/Sequent_calculus#Example_derivations) page? I need to write proofs of the same form for an assignment and would like a pointer as to how this could be implemented.

Greetings

**Comment by jamesrswift** [comment, marked as answer]
Would this do the trick? https://github.com/typst/packages/blob/a8567120a900e4325efb650f9e78e19f11d10608/packages/preview/prooftree/0.1.0/README.md

**Reply by AurelWeinhold** [reply]
Yes, this is exactly what I imagined. I thought I had checked all the packages. Thank you!

**Reply by pauladam94** [reply]
I was actually trying to rewrite this package my self. Thanks you so much. To this day this package is not present in the packages of the typst documention. It should be !

**Reply by jamesrswift** [reply]
It is? https://typst.app/docs/packages/

**Reply by pauladam94** [reply]
You are right ! I checked the packages a bit too fast.

**Reply by pauladam94** [reply]
What is the best way to participate to this package (directly in the typst/package ?) ? I think I have fixed some corner cases.

**Reply by laurmaedje** [reply]
You can make an issue or PR here: https://github.com/david-davies/typst-prooftree

You can find a package's repo / homepage by clicking on the chain icon in the package list.
